src/Components/PostingView/index.js

```diff
@@ -72,14 +82,24 @@ const PostingView = ({ eventId, getUserInfo }) => {
             </div>
             <div className="posting-position-cards-wrapper">
               {openJobs.map((job) => (
-                <section onClick={() => setChosenJob(job)} key={job.id} className="posting-positions-card">
+                <section
+                  onClick={() => setChosenJob(job)}
+                  key={job.id}
+                  className="posting-positions-card"
+                >
                   <h3>{job.name}</h3>
                   <p>Open Spots: {job.numberOfSpots}</p>
                 </section>
               ))}
             </div>
             <div className="submit-button-wrapper">
-              <button onClick={substractOpenPosition} className="submit-button">SUBMIT</button>
+              <button
+                onClick={substractOpenPosition}
```

Comment:
good feature! It's nice to show users when they are not allowed to choose more position by disabling the button. 
We might show specific effect/message that shows the confirmation of the position and that user already signed for this event.

Reply:
good idea! and add affect to reflect selection